src/go/README.md

```diff
@@ -0,0 +1,25 @@
+# Go APIView token file generator
+
+## Getting started
+
+### Prerequisites
+
+- Go 1.13 or above.
+
+### Generate the tool
+
+To create the command (this will output an executable), run in the `/go` directory:
+```
+go build
+```
+
+NOTE: To access the command anywhere you can add it to your $PATH variable. 
+
+### Run the tool
+
+Output a JSON file with the tokenized output for the SDK by running:
```

Comment:
Run the following command to generate the <filename> file containing the tokenized output for the SDK.

Reply:
Done